src/components/Layouts/Primary/TopBar/Items/ProfileMenu/ProfileMenu.test.tsx

```diff
@@ -65,6 +65,8 @@ const TestComponent: React.FC<TestComponentProps> = ({
   </ThemeProvider>
 );
 
+global.innerWidth = 1200;
```

Comment:
What is this for? Could it be moved inside of a `beforeAll` block?

Reply:
This is needed to make sure the screen in the test is large enough to show all the data that it is testing for. On smaller screens, the Profile name is not displayed. It could go in a beforeAll() but then I would have to make a parent describe block that wraps around all the other describe blocks. Should I do that?

Reply:
`beforeAll` can be outside of `describe` blocks. This seems to work.

```ts
beforeAll(() => {
  // Make the screen wide enough to show all data
  global.innerWidth = 1200;
});

describe('...');
```

Reply:
Oh nice. I didn't realize that